src/coprocessor/codec/mysql/time/mod.rs

```diff
@@ -314,6 +319,12 @@ impl Time {
         Time::new(time, mysql::types::DATETIME as u8, fsp as i8)
     }
 
+    pub fn parse_fsp(s: &str) -> i8 {
```

Comment:
Would you like to name it as `get_fsp` to be aligned with TiDB?

Reply:
Your comment is appreciated!

`Time` defined a `get_fsp` method(https://github.com/tikv/tikv/blob/036a1d265e4b029486191d4239753b4b0ce1bf43/src/coprocessor/codec/mysql/time/mod.rs#L179). I think we'd change the function name, or move this function to another module, what do you suggest? @breeswish